HospitalFrontend/src/views/registration/PersonalInformation.vue

```diff
@@ -0,0 +1,173 @@
+                    <option value="Widowed">Widowed</option>
+                    <option value="Divorced">Divorced</option>
+                </select>
+                <select name="gender" required v-model="personalInformation.Gender">
+                    <option value="" disabled selected>Gender</option>
+                    <option value="Male">Male</option>
+                    <option value="Female">Female</option>
+                    <option value="Other">Other</option>
+                </select>
+                 <select name="rescountry" required v-on:change="fetchCityOfResidence">
+                    <option value="" disabled selected>Country of residence</option>
+                    <option v-bind:value="country.id" v-for="(country,index) in countries" v-bind:key="index">{{country.name}}</option>
+                </select>
+                <select name="rescity" required v-if="!cityOfResidenceDisabled" v-on:change="bindCityOfResidence">
+                    <option value="" disabled selected>City of residence</option>
+                    <option v-bind:value="city.id" v-for="(city,index) in citiesOfResidence" v-bind:key="index">{{city.name}}</option>
+                </select>
+                <input type="text" name="address" placeholder="Address" v-model="personalInformation.Address" />
+                <select name="birthcountry" v-on:change="fetchCityOfBirth" required>
+                    <option value="" disabled selected>Country of birth</option>
+                    <option v-bind:value="country.id" v-for="(country,index) in countries" v-bind:key="index">{{country.name}}</option>
+                </select>
+                <select name="birthcity" required v-if="!cityOfBirthDisabled" v-on:change="bindCityOfBirth">
+                    <option value="" disabled selected>City of birth</option>
+                    <option v-bind:value="city.id" v-for="(city,index) in citiesOfBirth" v-bind:key="index">{{city.name}}</option>
+                </select>
+                <!--div class="dropdown">
+                    <button class="btn  dropdown-toggle" type="button" id="citizenshipsMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                        Citizenships
+                    </button>
+                    <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+                        <a class="dropdown-item row" >
+                            <div class="col">
+                                <input type="checkbox" aria-label="Checkbox for following text input" class="checkbox">
+                            </div>
+                            <div class="col">
+                                <h4>action</h4>
+                            </div>
+                        </a>
+                        <a class="dropdown-item">Another action</a>
+                        <a class="dropdown-item">Something else here</a>
+                    </div>
+                </div-->
```

Comment:
Why is this under comment?

Reply:
Because I was struggling to implement it properly, I'll delete it.